api/nodes/serializers.py

```diff
@@ -150,13 +151,17 @@ def create(self, validated_data):
         auth = Auth(user)
         node = self.context['view'].get_node()
         pointer_node = Node.load(validated_data['node']['_id'])
-        pointer = node.add_pointer(pointer_node, auth, save=True)
-        return pointer
+        if not pointer_node:
+            raise exceptions.NotFound('Node not found.')
```

Comment:
Oops, previous comment got lost with the diff--reposting...

On second thought, a 404 might not be the most appropriate since the the missing resource isn't the node being requested--it's the node being added as a pointer.

Maybe a 400 or a 422? ping @brianjgeiger